roles/sugarizer/tasks/main.yml

```diff
@@ -11,14 +11,23 @@
         state=link
 
 - name: Install sugarizer required packages
-  package: name={{ item }} 
+  package: name=nodejs
            state=present
-  with_items:
-    - nodejs
-    - npm
+#    - npm
   when: internet_available 
-  tags:
-    - download
+
+- name: Install npm non debian
+  package: name=npm
+           state=present
+  when: internet_available and not is_debian
+
+- name: Install npm on debian -- set up apt sources
+  shell: curl -sL https://deb.nodesource.com/setup_6.x | bash -
+  when: internet_available and is_debian
+
+- name: Actually get it installed
+  command: apt install -y npm
+  when: internet_available and is_debian
 
```

Comment:
Cannot imagine that a newer version of npm will fail on debian8 -- but I don't think I circled back to test -- will do that shortly